src/app/access-requests/services/access-request-field-edit.service.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Access Request Field Edit Service
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+/* eslint-disable boundaries/element-types */
+import { inject, Injectable } from '@angular/core';
+import { NotificationService } from '@app/shared/services/notification.service';
+import {
+  EditableFieldInfo,
+  EditActionType,
+} from '../features/access-request-manager-dialog/access-request-manager-dialog.component';
+import { AccessRequest } from '../models/access-requests';
+import { RemoveUrlFromTicketId } from '../pipes/remove-url-from-ticket-id.pipe';
+import { AccessRequestService } from './access-request.service';
+
+/**
+ * Handles the editing of access request fields.
+ */
+@Injectable({ providedIn: 'root' })
+export class AccessRequestFieldEditService {
+  #accessRequestService = inject(AccessRequestService);
+  #parseTicketId = inject(RemoveUrlFromTicketId);
+  #notificationService = inject(NotificationService);
+
+  /**
+   * Handles the editing of certain fields in the access request manager dialog.
+   * @param action The action to perform as a string literal with options 'show', 'cancel', or 'save'.
+   * @param editDictionary The dictionary object of the field being edited.
+   * @param ar The access request to edit.
+   */
+  handleEdit(
+    action: EditActionType,
+    editDictionary: EditableFieldInfo,
+    ar: AccessRequest,
+  ) {
+    switch (action) {
+      case 'show':
+        editDictionary.show = true;
```

Comment:
This is another source of NG0100 errors.

Here you edit the value of the signal in place. Instead, you need to use signal.set or signal.update to update it.

There are also some other places in this method where you do this.

Reply:
I.e. you need to pass the model, not the value.

Reply:
Fixed in 16e8e66.